mycroft/util/format.py

```diff
@@ -142,11 +142,16 @@ def date_format(self, dt, lang, now):
                 if dt.month == now.month and dt.day > now.day:
                     format_str = 'date_full_no_year_month'
 
-            if (now + datetime.timedelta(1)).day == dt.day:
+            tomorrow = now + datetime.timedelta(1)
```

Comment:
For readability maybe specify datetime.timedelta(days=1)

Reply:
Done